src/Hazelcast.Net.Examples/SoakTests/Soak1.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Hazelcast.DistributedObjects;
+using Hazelcast.Examples.Models;
+using Hazelcast.Query;
+using Microsoft.Extensions.Logging;
+
+namespace Hazelcast.Examples.SoakTests
+{
+    // run the soak test with:
+    //
+    // ./hz.ps1 run-example ~Soak1
+    //
+    // or, with arguments:
+    //
+    // ./hz.ps1 run-example ~Soak1 ---
+    //   --hazelcast:examples:soak1:ThreadCount=1             (valid integer)
+    //   --hazelcast:examples:soak1:Duration="HH:MM:SS"       (valid timespan)
+    //   --hazelcast:networking:addresses:0=127.0.0.1:5701
+    //   --hazelcast:networking:addresses:1=127.0.0.1:5702
+    //
+    // NOTE1: if you do not pass all arguments in one unique line, but use instead multiple lines
+    //  as displayed above, don't forget to terminate each line with the continuation character
+    //  for your shell, which is ` (backtick) for PowerShell or \ (backslash) for bash
+    // NOTE2: the --% thing is a PowerShell thing that is required for proper arguments parsing
+    //
+    // alternatively, to run from published examples, run with:
+    //
+    // ./hx.exe ~Soak1
+    //   --hazelcast:examples:soak1:ThreadCount=1             (valid integer)
+    //   --hazelcast:examples:soak1:Duration="HH:MM:SS"       (valid timespan)
+    //   --hazelcast:networking:addresses:0=127.0.0.1:5701
+    //   --hazelcast:networking:addresses:1=127.0.0.1:5702
+    //
+    // each server must run a 4.2+ server; start a server once by running:
+    //
+    // ./hz.ps1 run-server -server 4.2
+    //
+    // this will download all the required JARs from the proper repositories, *and*
+    // a default hazelcast-<version>.xml configuration file. stop the server, and
```

Comment:
Would be nice to remark the path. /build/hazelcast-<version>.xml

Reply:
yes, adding